etc/ld_wrapper.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# find the real linker, from env or same defaults as go
```

Comment:
this is one of those things that one looks at and thinks "ah yeah that's simple" until you know that this is the part that took the longest and looks simple because of all the work that went into it :)